dbmongo/js/RawDataTypes.ts

```diff
@@ -62,6 +62,7 @@ export type BatchValueProps = {
   effectif_ent: ParHash<EntréeEffectif>
   bdf: ParHash<EntréeBdf>
   diane: ParHash<EntréeDiane>
+  ellisphere: ParHash<Ellisphere>
```

Comment:
si ça te convient, je suggère qu'on renomme le type `Ellisphere` en `EntréeEllisphère`, pour cohérence avec les autres types:

```suggestion
  ellisphere: ParHash<EntréeEllisphere>
```